### ISSUE TITLE
Peer review from Ellen

### ISSUE THREAD

**Issue by ellenhsieh** [css-research/fp-KenChenCompEcon#4]
Thank you for presenting this interesting topic. It’s a really nice study on intergeneration mobility in China, which provides us some different insight towards intergeneration mobility. The outlay of the poster is neat. Also, the poster includes all the important parts of the study.

As for design of this poster, the use of UChicago red is really pretty. The design is simple but attracting. Also, in the background section, you highlighted the two objectives in the paragraph, which can help audience to get a quick idea on your research’s goal. However, it would be nice for data and method section to use  the bullet point to list out the important point. 

For the plots, the principal opponents section can simply explain the plots under them, instead of using a short paragraph, so that it would easier to know what the plots stand for. As for feature selection and PDP/ICE plots, it would be better if the name of the variable are named properly, not in their raw name. The transition matrix plot look really cool but it is kind of hard to see the corner part, maybe this plot would be nice if it can move and let people see the part conceal by the tall part (If it could be present on a webpage).

To sum up, this poster did a great job on conveying the topic. The design and contents are really nice. I really love your outlay and color for this poster. The conclusion is really clear and easy to understand.
